1.17/24.3.4/node/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/elixir:1.17.0-erlang-24.3.4
```

Comment:
Elixir 1.17.0 has a minimum requirement of OTP 25 https://github.com/elixir-lang/elixir/blob/v1.17/CHANGELOG.md#erlangotp-support

Appreciate the new version to be added - would it be possible to release an image with the latest OTP 26 also? That would be OTP 26.2.5 https://www.erlang.org/downloads

Reply:
We've also upgraded our apps to OTP 27 with 1.17 to get all the new features, so adding that as well would be great

Reply:
No major rush since we are using the main image in the meantime, but would be great to switch back to the circle images again